content/tutorials/custom-zk-chain/20.customizing-your-chain.md

```diff
@@ -0,0 +1,360 @@
+---
+title: Customizing your ZK chain
+description: Create a ZK chain with a custom ERC20 base token.
+
+---
+
+One way you can customize your ZK chain is by changing the base token used for gas from ETH to an ERC20 token.
+Let's try customizing a new chain by using a custom ERC20 base token.
+
+For the first release of the ZK stack, the base tokens you can use for your chain must be whitelisted.
+The whitelisted addresses are stored and checked in the `BridgeHub` contract on the L1.
+In the future it will be possible to add a new token in a permissionless proccess.
```

Comment:
This is the first mention of ZK stack in this ZK chain guide. The wording felt a little confusing as it sounded like I (as the reader going through the tutorial) am releasing a zk stack and have to whitelist. 

Do we mean more along the lines of:
```suggestion
With the initial release of ZK stack, base tokens must be whitelisted for use on chains.
```